conf/server.yaml.template

```diff
@@ -0,0 +1,24 @@
+SERVER:
+  # Server hostname
+  # HOSTNAME: replace-with-satellite-hostname
+  # Path to private ssh key to be used when connecting via SSH.
+  # SSH_KEY:
+  # HTTP scheme when building the server URL
+  # Suggested values for "scheme" are "http" and "https".
+  SCHEME: https
+  # Server port
+  PORT: 443
+  # SSH username
+  SSH_USERNAME: root
+  # SSH password if not using a SSH key to connect
+  # SSH_PASSWORD: replace-with-ssh-passwd
```

Comment:
Nitpick, but it would be good to organize these a bit better - keep the `SSH_*` keys together, and the `HOSTNAME`/`SCHEME`/`PORT` keys together

Reply:
Done , with comment on top of them :

`# Use either of SSH_PASSWORD or SSH_KEY or SSH_KEY_STRING below`